### ISSUE TITLE
"No input file specified" on fresh install

### ISSUE THREAD

**Issue by deviant77** [FreshRSS/FreshRSS#1843]
Hi!

I'm following the instructions [here](https://freshrss.github.io/FreshRSS/en/admins/02_Installation.html) - paying attention to the security warning to expose only the ./p/ directory on the web.

DocumentRoot is set to "/home/me/freshrss/p" (as it says in the documentation).

But when I visit the site, I am redirected to /i with the message "No input file specified". There's no installer.

If I set the DocumentRoot to "/home/me/freshrss" it works fine, but then I'm exposing all the directories.

**Comment by Alkarex** [comment]
Could you please give a bit more information about your system? (versions, server...)
And maybe paste the full configuration file

**Comment by deviant77** [comment]
Thanks for taking the time to reply.
The problem is sorted now.
It's a cPanel server. I'm not sure what the problem was, but I flipped the PHP version and it suddenly started working. I flipped the PHP version back to the original and it was still working. I even recreated the same problem again. So it might be a bug in cPanel.

Thanks for FreshRSS. I was looking for a new RSS client on Android to use with Inoreader, found FeedMe, which led me here. :-)
